.github/workflows/build-for-release.yml

```diff
@@ -9,37 +9,50 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        build_name:
+          [linux-x64-musl, linux-arm64-musl, macos-arm64, windows-x64-msvc]
         include:
-          - os: ubuntu-latest
+          - build_name: linux-x64-musl
+            os: ubuntu-latest
             exe_suffix: ""
             name_suffix: ""
             target: x86_64-unknown-linux-musl
-          - os: macos-latest
+          - build_name: linux-arm64-musl
+            os: ubuntu-latest
+            exe_suffix: ""
+            name_suffix: ""
+            target: aarch64-unknown-linux-musl
+          - build_name: macos-arm64
+            os: macos-latest
             exe_suffix: ""
             name_suffix: ".arm"
             target: aarch64-apple-darwin
-          - os: windows-latest
+          - build_name: windows-x64-msvc
+            os: windows-latest
             exe_suffix: ".exe"
             name_suffix: ".exe"
             target: x86_64-pc-windows-msvc
     steps:
       - uses: actions/checkout@v3
-      - name: Update rust
-        run: rustup update
-      - name: Add target
-        run: rustup target add ${{ matrix.target }}
-      - name: Cache dependencies
-        uses: Swatinem/rust-cache@v2
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
```

Comment:
`actions-rs` is unmaintained and the runners include rust by default, does it not work with `target add`?

Reply:
Reverted :3